index.js

```diff
@@ -1,23 +1,39 @@
-const { exec } = require('child_process');
+const {exec} = require('child_process');
 
 let curPid;
 let uids = {};
+// for windows support
+let curDir;
+const dirRegex = /([\w]:[\s\S]+)>/;
 
-const setCwd = (pid) =>
-  exec(`lsof -p ${pid} | grep cwd | tr -s ' ' | cut -d ' ' -f9-`, (err, cwd) => {
-    if (err) {
-      console.error(err);
-    } else {
-      cwd = cwd.trim();
+const setCwd = (pid) => {
+  if (process.platform === 'win32') {
```

Comment:
If you made the change described below, I think you might not need this anymore.